ui/desktop/src/ChatWindow.tsx

```diff
@@ -313,6 +313,33 @@ function ChatContent({
   );
 }
 
+// Function to send the system configuration to the server
+const addSystemConfig = async () => {
+  const systemConfig = {
+    type: "Stdio",
+    cmd: "goose",
```

Comment:
this requires that someone has goose installed with the latest on their system, or for developers something like 
```
export PATH=$HOME/Development/goose/target/release/:$PATH
```

We'll need to figure out a better approach for this for our builtin systems. It might be easiest to make them builtin to goosed library so we could use the same one we bundle with the app

Reply:
yeah I think will need to be built up with goose(d) into one binary for built in - is that possible? 

For adding others, will that be able to work as well without having to install goose seperately? (as that breaks the .app distribution model?)

Reply:
or if they are able to be pull down as their own binaries perhaps? (but if really built in, will want to have it bundled out of the box ideally, statically built is easiest, mostly as solving distribution things is probably more time consuming to get right on desktop apps)

Reply:
"It might be easiest to make them builtin to goosed library so we could use the same one we bundle with the app"

Yeah, this makes sense to me. We already have that binary inside the app, so we should use it.